src/components/Referrals/ReferralJourney/ReferralJourney.tsx

```diff
@@ -19,41 +19,61 @@ import {
   JourneyWrapper
 } from './ReferralJourney.styled'
 import { ANIMATION_DURATION } from './utils'
-import { AnimationPhaseType, ReferralJourneyProps } from './ReferralJourney.types'
+import { AnimationPhase as AnimationPhase, ReferralJourneyProps } from './ReferralJourney.types'
 
 // eslint-disable-next-line import/no-named-as-default-member
 const ReferralJourney = React.memo((props: ReferralJourneyProps) => {
-  const { invitedUsersAccepted } = props
-  const [animatedStep, setAnimatedStep] = useState(TIERS.filter(tier => tier.completed).length)
+  const { invitedUsersAccepted, onSetReferralEmail } = props
+  const [animatedStep, setAnimatedStep] = useState(TIERS.filter(tier => tier.invitesAccepted <= invitedUsersAccepted).length)
   const [open, setOpen] = useState(false)
-  const [journeyTiers, setJourneyTiers] = useState(TIERS)
-  const [animationPhase, setAnimationPhase] = useState<AnimationPhaseType>(AnimationPhaseType.WAITING_NEXT_TIER)
+  const [journeyTiers, setJourneyTiers] = useState(
+    TIERS.map(tier => ({ ...tier, completed: tier.invitesAccepted <= invitedUsersAccepted }))
+  )
+  const [animationPhase, setAnimationPhase] = useState<AnimationPhase>(AnimationPhase.WAITING_NEXT_TIER)
   const totalSteps = TIERS.length
   const journeyStepRefs = useRef<(Element | null)[]>([])
   const sectionRef = useRef<HTMLDivElement>(null)
 
+  const tooltipTitle = useMemo(() => {
+    const currentTierIndex = TIERS.findIndex(tier => invitedUsersAccepted < tier.invitesAccepted)
+    const nextTier = currentTierIndex !== -1 ? TIERS[currentTierIndex] : null
```

Comment:
If you want, you can simplify this by using `find`:
```suggestion
    const nextTier = TIERS.find(tier => invitedUsersAccepted < tier.invitesAccepted) ?? null
```